emily/handler/src/api/models/deposit/responses.rs

```diff
@@ -30,5 +30,15 @@ pub struct GetDepositsResponse {
 #[serde(rename_all = "camelCase")]
 pub struct UpdateDepositsResponse {
     /// Deposit infos: deposits with a little less data.
-    pub deposits: Vec<Deposit>,
+    pub deposits: Vec<DepositWithStatus>,
+}
+
+/// Wrapper for deposit with status code. Used for multi-status responses.
+#[derive(Clone, Default, Debug, PartialEq, Hash, Serialize, Deserialize, ToSchema, ToResponse)]
+#[serde(rename_all = "camelCase")]
+pub struct DepositWithStatus {
+    /// Deposit itself.
+    pub deposit: Deposit,
+    /// Status code.
```

Comment:
```suggestion
    /// The status code of update operation associated with the
    /// above deposit request.
```

Reply:
Hmm, tbh I don't really like your wording but I agree we should annotate that it is http status, not sbtc (`Pending`, `Accepted`, etc) status